applications/StructuralMechanicsApplication/custom_constitutive/elastic_isotropic_3d.cpp

```diff
@@ -60,17 +60,19 @@ void  ElasticIsotropic3D::CalculateMaterialResponsePK2(ConstitutiveLaw::Paramete
     KRATOS_TRY;
     // b.- Get Values to compute the constitutive law:
     Flags & r_constitutive_law_options = rValues.GetOptions();
-
     Vector& r_strain_vector = rValues.GetStrainVector();
 
     //NOTE: SINCE THE ELEMENT IS IN SMALL STRAINS WE CAN USE ANY STRAIN MEASURE. HERE EMPLOYING THE CAUCHY_GREEN
     if( r_constitutive_law_options.IsNot( ConstitutiveLaw::USE_ELEMENT_PROVIDED_STRAIN )) {
         CalculateCauchyGreenStrain( rValues, r_strain_vector);
     }
 
+    this->AddInitialStrainVectorContribution( r_strain_vector, rValues );
```

Comment:
do you need the "this"?

Reply:
nope, I can remove it -> BTW, this strain calculation has to be modified in the future. The correct way of doing it is to call the CalculateMaterialResponse deactivating th calculate stress and D.